go/README.md

```diff
@@ -0,0 +1,69 @@
+# GLOME-Go
+**This is not an officially supported Google product.**
+
+This repository contains a Golang implementation for the GLOME protocol. You can
+find the library as well as the tests on the folder glome. 
+
+
+## Python API
+
+### Requirements
+
+- golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
+
+### Example
+
+We provide a brief example of use.  In order for Alice and Bob to communicate,
+the first step would be to generate some new keys:
+
+```go
+import (
+        "glome" 
+        "crypto/rand"
+)
+
+// Alice generates new random KeyPair
+alicePub, alicePriv, err := glome.GenerateKeys(rand.Reader)
+
+// Bob Generates Private Key from as existing byte array
+b := [32]byte{0,2,...,7,6}
+bobPriv := glome.PrivateKey(b)
+
+// Bob could have as well generate the key from byte slice
+s := b[:]
+bobPriv, _ := glome.PrivateKeyFromSlice(s)
+
+// Bob deduces publi key
+bobPub := bobPriv.Public()
+```
+
+Suppose that Alice knows `bobPub` and wants to send Bob the message
+`msg` and no other message have been shared before. Alice will need to generate 
+a `Dialog`
+
+```go
+ex := alicePriv.Exchange(bobPub)
+firstTag = ex.Tag(msg, 0)
+```
+
+And Alice will send Bob both `msg`, `firstTag` as well as Alice's public key.
+On Bob ends he will need to do the following:
+
+```go
+ex := alicePriv.Exchange(bobPub)
```

Comment:
Should this be `bobPriv.Exchange(alicePub)`?

Reply:
Yes, it should be! fixed.